Add receptorctl work cancel doc and doc lint make targets (#1055)
ansible/receptor: docs/source/receptorctl/receptorctl_work_cancel.rst

-----------
work cancel
-----------

.. contents::
   :local:

> ``receptorctl work cancel`` cancels (kills) one or more units of work.

Reviewer: ```suggestion
``receptorctl work cancel`` kills one or more units of work.
```

Reviewer: I'd say it's better not to repeat the word "cancel" here. Consider "stops" or "terminates" as an alternative to "kills" - but I think it's fine given there is the standard `kill` command.

In the work unit docs the `canceled` state uses the word "terminate": https://github.com/ansible/receptor/blob/devel/docs/source/receptorctl/receptorctl_work_list.rst#work-unit-states

Command syntax: ``receptorctl --socket=<socket_path> work cancel <<Unit ID>> [...]``

``socket_path`` is the control socket address for the Receptor connection.
   The default is ``unix:`` for a Unix socket.
   Use ``tcp://`` for a TCP socket.
   The corresponding environment variable is ``RECEPTORCTL_SOCKET``.

.. code-block:: text

  ss --listening --processes --unix 'src = unix:<socket_path>'
  Netid         State          Recv-Q         Send-Q                   Local Address:Port                     Peer Address:Port        Process
  u_str         LISTEN         0              4096                   /tmp/local.sock 38130170                            * 0            users:(("receptor",pid=3226769,fd=7))

``ps -fp $(pidof receptor)``
``lsof -p <pid>``

> ``Unit ID`` is a unique identifier for a work unit (job).  This should be the local Unit ID for the Receptor instance you are connected to,

Reviewer: ```suggestion
``Unit ID`` is a unique identifier for a work unit (job). When running the ``work cancel`` command, you should specify the ``Unit ID`` for the Receptor instance to which you are connected.
```

> ``Unit ID`` is a unique identifier for a work unit (job).  When running the ``work cancel``command, you should specify the ``Unit ID`` for the Receptor instance to which you are connected.

Reviewer: ```suggestion
``Unit ID`` is a unique identifier for a work unit (job).  When running the ``work cancel`` command, you should specify the ``Unit ID`` for the Receptor instance to which you are connected.
```
